ci/test-checks.sh

```diff
@@ -46,6 +46,8 @@ echo --- build environment
   cargo audit --version
 
   grcov --version
+
+  /home/.cargo/bin/sccache --version
```

Comment:
I think we can do something like:

```suggestion
  if [[ -v RUSTC_WRAPPER ]]; then
    $RUSTC_WRAPPER --version
  elif command -v sccache >/dev/null 2>&1; then
    sccache --version
  else
    echo "sccache doesn't exist"
  fi
```

so that we can run this script in
- docker environment
- non-docker environment (btw, this one need some other PRs to succeed)
- local environment

Reply:
hmm, actually, i'd rather this to be dead-simple like `grcov --version` just above. btw, why `sccache` isn't installed inside docker image like `grcov`?

Reply:
hint: i'd like sneak `cargo-udeps` in the image: ref: #31320

Reply:
let me try to install it into our docker image. iirc I didn't install it cuz we copied the whole cargo dir into image. maybe I just think I can just use it 🤔 

I will try to install it and bring you back 😈